src/Corrade/Utility/Test/DirectoryTest.cpp

```diff
@@ -585,12 +587,37 @@ void DirectoryTest::mkpathNoPermission() {
     if(Directory::exists("/nope"))
         CORRADE_SKIP("Can't test because the destination might be writeable");
 
-    CORRADE_VERIFY(!Directory::mkpath("/nope/never"));
+    std::ostringstream out;
+    Error err{&out};
+    {
+        /* Ensure errors are printed in English */
+        const char* currentLocale = std::setlocale(LC_ALL, nullptr);
+        std::setlocale(LC_ALL, "en_US.utf8");
+        ScopeGuard restoreLocale{currentLocale, [](const char* locale) {
+            std::setlocale(LC_ALL, locale);
+        }};
+
+        CORRADE_VERIFY(!Directory::mkpath("/nope/never"));
+    }
+
+    CORRADE_COMPARE(out.str(),
+        "Utility::Directory::mkpath(): error creating /nope: Permission denied\n");
+
     #else
     if(Directory::exists("W:/"))
         CORRADE_SKIP("Can't test because the destination might be writeable");
 
+    std::ostringstream out;
+    Error err{&out};
+
     CORRADE_VERIFY(!Directory::mkpath("W:/nope"));
+
+    /* On Windows we cannot enusre messages are printed in a certain
+     * language, as they depend on the user's installed languages */
+    const std::string output = out.str();
+    const size_t newlinePos = output.find('\n');
+    CORRADE_VERIFY(newlinePos != std::string::npos);
+    CORRADE_COMPARE(output.substr(0, 49), "Utility::Directory::mkpath(): error creating W:: ");
```

Comment:
Not sure if the first two lines are still relevant... maybe instead do

```suggestion
    CORRADE_COMPARE(output.substr(0, 49), "Utility::Directory::mkpath(): error creating W:: ");
    /* Check that there's just one newline at the end, not two */
    CORRADE_COMPARE(output.find('\n'), output.size() - 1);
```

Reply:
Yeah, good catch, they are not.

Reply:
✅